src/app/services/properties/properties-wishlist/properties-wishlist.service.spec.ts

```diff
@@ -0,0 +1,122 @@
+import { TestBed, tick } from '@angular/core/testing';
+
+import { PropertiesWishlistService } from './properties-wishlist.service';
+import { HttpTestingController, HttpClientTestingModule } from '@angular/common/http/testing';
+import { stringify } from '@angular/compiler/src/util';
+import { APPCONFIG } from 'src/app/config';
+
+describe('test for PropertiesWishlistService', () => {
+  let httpTestingController: HttpTestingController;
+  let service: PropertiesWishlistService ;
+  let wishlistUrl : string;
+  const testSlug = 'test-test'
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [PropertiesWishlistService],
+      imports: [HttpClientTestingModule]
+    });
+
+    httpTestingController = TestBed.get(HttpTestingController)
+    service = TestBed.get(PropertiesWishlistService);
+    wishlistUrl = `${APPCONFIG.base_url}${service.wishlistUrl}`;
+  });
+
+  afterEach(() => {
+    httpTestingController.verify();
+  });
+
+  it('should be created', () => {
+    expect(service).toBeTruthy();
+  });
+
+  it('should return all items in my wishlist', ()=>{
+    const mockWihslistProperties = {
+      "data": {
+        "property": [
+          {
+            "id": 7,
+            "price": 20000000.0,
+            "title": "Gemini Court",
+            "property_type": "B",
+            "description": "Awesome property",
+            "slug": "g-c"
+          },]}};
+   
+    service.getMyWishlist().subscribe(response=>{
+      const returnedProperties = response.data.property;
+      expect(returnedProperties[0].id).toEqual(7);
+      expect(returnedProperties[0].title).toEqual("Gemini Court");
+      expect(returnedProperties[0].slug).toEqual("g-c");
+
+    })
```

Comment:
we are missing a semicolon here
```suggestion
    });
```

Reply:
Done